.github/workflows/deploy-infra.yaml

```diff
@@ -497,3 +497,19 @@ jobs:
               --group read-only \
               --username $user
           done < "infra/cluster_users"
+
+  report-if-failed:
+    name: Report if workflow failed
+    runs-on: ubuntu-20.04
+    needs: [setup, create-eks-cluster, configure-cluster]
+    if: failure()
+    steps:
+      - id: send-to-slack
+        name: Send failure notification to Slack on failure
+        env:
```

Comment:
same here -- we want to see failures if they are with the master branch. If they are not with the master branch, most likely  a developer is testing and then the notification will just be noise

Reply:
Same as above. If a developer is testing, they'll pay attention to the run.